src/commonMain/kotlin/com/charleskorn/kaml/YamlConfiguration.kt

```diff
@@ -43,6 +43,7 @@ public data class YamlConfiguration constructor(
     internal val sequenceStyle: SequenceStyle = SequenceStyle.Block,
     internal val singleLineStringStyle: SingleLineStringStyle = SingleLineStringStyle.DoubleQuoted,
     internal val multiLineStringStyle: MultiLineStringStyle = singleLineStringStyle.multiLineStringStyle,
+    internal val ignoreEmptyDocument: Boolean = false,
```

Comment:
This name might be a little clearer:

```suggestion
    internal val allowReadingEmptyDocument: Boolean = false,
```